src/components/organisms/Navbar/Navbar/Navbar.tsx

```diff
@@ -267,6 +267,14 @@ const SmallDeviceNavbar = styled.div`
   `}
 `;
 
+const NavItemsWrapper = styled.ul`
+  padding-left: 0;
+
+  > li {
```

Comment:
This component needs to be `inline-block` for the desktop view, targeting via parent is the only way without splitting the two components which are otherwise very similar.